viewing_party/main.py

```diff
@@ -0,0 +1,157 @@
+def create_movie(movie_title,genre,rating):
+
+    new_movie={
+        "title":movie_title,
+        "genre":genre,
+        "rating":rating
+    }
+
+    for values in new_movie.values():
+      if values==None:
+        return None
+      
+    return new_movie
```

Comment:
check if parameter data is valid FIRST, then program doesn't have to run code that isn't needed.

```suggestion
if movie_title and genre and rating:
    new_movie={
        "title":movie_title,
        "genre":genre,
        "rating":rating
    }
   return new_movie
else:
     return None

```